.github/workflows/application.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["7.4", "8.0"]
-                symfony: ["^4.4", "5.2.*", "5.3.*"]
+                symfony: ["^4.4", "5.2.*", "5.3.*", "5.4.*"]
```

Comment:
What's the point of testing 5.2 & 5.3 if you force requirements to be: `^4.4. || ^5.4`?

Reply:
Fixed on package level 👍